packages/cli/src/constructs/playwright-check.ts

```diff
@@ -159,8 +197,22 @@ export class PlaywrightCheck extends RuntimeCheck {
       ))
     }
 
+    this.#validateGroupReferences(diagnostics)
+  }
+
+  #validateGroupReferences (diagnostics: Diagnostics): void {
+    if (this.groupId) {
+      const checkGroup = PlaywrightCheck.#findGroupByRef(this.groupId)
+      if (!checkGroup) {
+        diagnostics.add(new InvalidPropertyValueDiagnostic(
+          'groupId',
+          new Error(`No such group with ID "${this.groupId.ref}".`),
```

Comment:
This can result in `No such group with ID "undefined".`  if `groupId` is a string. But again, groupId is not what we want (though it should be supported).